addons/account/models/account_move_line.py

```diff
@@ -564,17 +564,17 @@ def _compute_account_id(self):
                 )[0]
         for line in self:
             if not line.account_id and line.display_type not in ('line_section', 'line_note'):
-                line.account_id = line.company_id.account_journal_suspense_account_id
+                previous_two_accounts = line.move_id.line_ids.filtered('account_id')[-2:].account_id
+                if len(previous_two_accounts) == 1:
+                    line.account_id = previous_two_accounts
+                else:
+                    line.account_id = line.company_id.account_journal_suspense_account_id
```

Comment:
I don't understand why you put the suspense account here. It's supposed to be used only by statement lines, right?

Reply:
No, it is used anywhere you don't know which account to put, when you need to create an entry

A next spec will come to maybe not make the account required as long as the entry is not posted